package.json

```diff
@@ -9,12 +9,12 @@
     "build:date-adapters": "tsc -p tsconfig-date-adapters.json",
     "build:styles": "node-sass projects/angular-calendar/src/angular-calendar.scss | postcss --output dist/angular-calendar/css/angular-calendar.css",
     "build:clean": "del-cli dist",
-    "build:schematics": "tsc projects/angular-calendar/schematics/ng-add/index.ts",
+    "build:schematics": "tsc --skipLibCheck projects/angular-calendar/schematics/ng-add/index.ts",
```

Comment:
Do we need `--skipLibCheck` here? I removed it locally when playing with the repo and it all seemed to work ok without it